core/feature-manager/org.wso2.carbon.feature.mgt.core/pom.xml

```diff
@@ -44,7 +44,7 @@
             <artifactId>org.eclipse.equinox.p2.artifact.repository</artifactId>
```

Comment:
Why exclude "org.eclipse.platform", This version of org.eclipse.equinox.p2.artifact.repository wouldn't have a dependency to *.platform. Check the transitive dependency and which one is the correct one to exclude.